postr/youtube_postr.py

```diff
@@ -103,10 +151,8 @@ def channels_list_by_id(client: Any, **kwargs: str) -> Any:
 
     return response
 
-# TODO type ignore fix
-
 
-def generate_build(credentials) -> Any:  # type: ignore
+def generate_build(credentials: Any) -> Any:
```

Comment:
why is credentials an Any? what actually is it?

Reply:
credentials are OAuth credentials. I have since added the typing in my new commit. Thanks.